packages/runner/lib/server.ts

```diff
@@ -58,14 +67,51 @@ function runProcedure() {
                     input: codeParams
                 });
                 usage.track(nangoProps);
-                return await exec(nangoProps, input.isInvokedImmediately, input.isWebhook, code, codeParams);
+                const scriptType: 'sync' | 'action' | 'webhook' = input.isWebhook ? 'webhook' : input.isInvokedImmediately ? 'action' : 'sync';
+                return await exec(nangoProps, scriptType, code, codeParams);
             } finally {
```

Comment:
this type specification is a bit messy but is this temporary since it is combining the two?

Reply:
The `runProcedure` and therefore the awkward scriptType transformation will go away once jobs is refactored